signing/sawtooth_signing/secp256k1_signer.py

```diff
@@ -184,7 +111,7 @@ def _decode_pubkey(serialized_pubkey, encoding_format='hex'):
 
 
 def generate_identifier(pubkey):
```

Comment:
Is this method used any more?  I thought this was a vestige of the bitcoin tools.